src/hooks/useBindingSelectorCells.tsx

```diff
@@ -0,0 +1,161 @@
+                    resetCollectionMetadata([], rows);
+                }
+
+                const publishedCollections =
+                    hasLength(discoveredCollections) && hasLength(collections)
+                        ? collections.filter(
+                              (collection) =>
+                                  !discoveredCollections.includes(collection)
+                          )
+                        : [];
+
+                if (draftId && publishedCollections.length > 0) {
+                    void deleteDraftSpecsByCatalogName(
+                        draftId,
+                        'collection',
+                        publishedCollections
+                    );
+                }
+            },
+            toggleCollections: (rows: any[] | null, value: boolean) =>
+                toggleCollections(rows, value),
+        }),
+        [
+            collections,
+            discoveredCollections,
+            draftId,
+            removeBindings,
+            resetCollectionMetadata,
+            task,
+            toggleCollections,
+            workflow,
+        ]
+    );
+
+    // TODO (collection selector)
+    //  We should make this return an object where the keys match the `COLLECTION_SELECTOR_...` props set as `field` on the columns
+    return useMemo(() => {
+        const cellRenderers: CollectionSelectorCellSettings = {
+            name: {
+                cellRenderer: (params, filterValue) => {
+                    const filteringActive = Boolean(filterValue);
+
+                    const collectionParts = filteringActive
+                        ? splitPathAndName(
+                              params.row[COLLECTION_SELECTOR_NAME_COL]
+                          )
+                        : isCapture
+                          ? [params.row[COLLECTION_SELECTOR_STRIPPED_PATH_NAME]]
+                          : [params.row[COLLECTION_SELECTOR_NAME_COL]];
+
+                    return (
+                        <BindingsSelectorName
+                            collection={collectionParts}
+                            filterValue={filterValue}
+                            buttonProps={{
+                                startIcon: isCollection ? undefined : (
+                                    <BindingsSelectorErrorIndicator
+                                        bindingUUID={
+                                            params.row[
+                                                COLLECTION_SELECTOR_UUID_COL
+                                            ]
+                                        }
+                                        collection={
+                                            params.row[
+                                                COLLECTION_SELECTOR_NAME_COL
+                                            ]
+                                        }
+                                    />
+                                ),
```

Comment:
Allow passing in the start icon so that transformation create won't show an error indicator that has no details.